.github/workflows/ruby.yml

```diff
@@ -14,12 +14,6 @@ jobs:
           - macos
           - windows
         ruby:
-          - 2.1
-          - 2.2
-          - 2.3
-          - 2.4
-          - 2.5
-          - 2.6
```

Comment:
Can you please revert this bit?  As long as it's not insurmountable, we try to maintain working compatibility for as far back as possible.  It would be nice to know if/when it broke on older versions so we can fix it if possible instead of dropping support.